index.html

```diff
@@ -1,13 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>"ALL about me"</title>
+    <meta name="Farida" content="HTML"/>
+  </head>
+  <body>
 <h1>Farida Hyderi</h1>
+    <section id="About">
 <h2>About</h2>
 <p>My name is Farida I am very entusiastic to learn new programming langusges
 also very much interested in testing </p>
+    </section>
+
+    <section id = "Experience">
 <h2>Experience</h2>
-Cognizant Agile Methodology from Forage
-Accenture Project management Forage
-Mastercard - Cybersecurity Job Simulation from Forage
-Worked on testing projects
+<li>Cognizant Agile Methodology from Forage </li>
+<li>Accenture Project management Forage </li>
+<li>Mastercard - Cybersecurity Job Simulation from Forage</li>
+<li>Worked on testing projects</li>
+    </section>
+
+    <section id = "connect">
 <h2>Connect</h2>
-<li><www.linkedin.com/in/farida-hyderi-a32700226</www.linkedin.com</li>
-<li>gmail.com</li>
-<li>instagram</li>
+<a href="www.linkedin.com/in/farida-hyderi-a32700226</www.linkedin.com"> </a>
+<a href="gmail.com"</a>
+<a href="instagram"</a>
```

Comment:
Good work adding the web addresses you're linking to. Currently though, the web addresses aren't attached to any text. For them to show up, you'll need to add text. For example, 
```
<a href="instagram"> My Instagram</a>
```
Have a look at [this reference for more info](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)